Makefile

```diff
@@ -713,6 +736,8 @@ $(BUILD_DIR)/src/code/z_actor_dlftbls.o: include/tables/actor_table.h
 $(BUILD_DIR)/src/code/z_effect_soft_sprite_dlftbls.o: include/tables/effect_ss_table.h
 $(BUILD_DIR)/src/code/z_game_dlftbls.o: include/tables/gamestate_table.h
 $(BUILD_DIR)/src/code/z_scene_table.o: include/tables/scene_table.h include/tables/entrance_table.h
+$(BUILD_DIR)/src/audio/general.o: $(SEQUENCE_TABLE) include/tables/sfx/*.h
```

Comment:
I think we should hardcode include/tables/sequence_table.h too, it seems inconsistent that this gets its own variable for some reason

Reply:
It's used all over the place so it seems nicer to define it in a variable rather than copy the path everywhere, even if the path isn't expected to change

Reply:
eh, ultimately it's not too important, but I was a bit annoyed I had to scroll up to figure out what the path was, and I think it makes this section less beautiful because it sticks out